courses/fundamentals_of_ada/180_polymorphism.rst

```diff
@@ -39,9 +39,9 @@ Examples
 Classes 
 ----------
 
-* In Ada, a Class denotes an inheritance subtree
-* Class of `T` is the class of `T` and all its children
-* Type `T'Class` can designate any object typed after type of class of `T`
+* In Ada, a Class denotes a tagged inheritance subtree
+* Class of `T` is `T` and all its children
```

Comment:
Probably should be :ada:`T` (throughout file)

Reply:
Agreed